### ISSUE TITLE
Suggestion (Pokemons not to catch): Catch and transfer instead of skipping.

### ISSUE THREAD

**Issue by Caydenz** [NecronomiconCoding/NecroBot#613]
It would be a lot better to catch the pokemon and transfer it afterwards instead of just skipping it.

**Comment by kanfkasndklasdn** [comment]
This is what it's doing when you don't set the variable to true. Setting the variable to true skip those pokemon in order to catch quicklyer other pokemon so you get more candies in those.

**Comment by Caydenz** [comment]
It's not doing that. If the CP or IV is higher than the threshold, the pokemon won't be transferred.

**Comment by agp89** [comment]
So what you mean is like a list "PokemonsToTransfer" which does not respect the CP or IV threshold but still respects "KeepMinDuplicatePokemon"?

**Comment by Caydenz** [comment]
Yep, that's what I mean. Something like that will be fine as well, probably even better.

**Comment by srounet** [comment]
Same thing here:

    Encounter problem: PokemonInventoryFull

There are pokemons that you know, you don't want to keep, but still want to catch for xp.
So having a File where you can specify the names of the pokemons you want to "throw" no matter what CP or IV it is would be nice.
It would be the exact opposite of the "Mew, Mewtwo", which obviously, you want to Keep no matter what :)

**Comment by salvob41** [comment]
This would be awesome!

**Comment by Wil32** [comment]
close #492 if this get completed. its basically the same thing

**Comment by bignuts** [comment]
if u set transfer duplicate pokemon to true and set it to 0, it does exactly what u mean, keeps only the one with IV and CP that meet requirements threshold.

We cannot remove pokemonnottocatch filter as a lot of us are using gpx pathing and if u don't filter some pokemon u're out of pokeball really fast!

**Comment by srounet** [comment]
That's true but then, if I have **35 * Pinsir of 2kCp and 95% IV** and still want to get rid of them how can I do without messing with my other really rare pokemons with high IV and CP?

There is a list of "MustKeepPokemon no matter what", there should be the opposite "TransferPokemon no matter what".
That way you can still capture them for XP, candies and throw them away just after.

**Comment by nenko91** [comment]
Good idea
